docs/service-discovery/dns.md

```diff
@@ -0,0 +1,44 @@
+# DNS Service Discovery
+
+DNS is a name resolution protocol used to determine IP addresses for hostnames.
+That makes it a natural fit for service discovery. Among others, Consul and AWS Cloud Map provide DNS APIs for service discovery.
+
+This page explains how Stork can use DNS to handle the service discovery.
+
+## Dependency
+
+First, you need to add the Stork Consul Service Discovery provider:
+
+```xml
+<dependency>
+    <groupId>io.smallrye.stork</groupId>
+    <artifactId>stork-service-discovery-dns</artifactId>
+    <version>{{version.current}}</version>
+</dependency>
+```
+
+## Configuration
+
+At the moment DNS service discovery in Stork does not support reading the port of the service from DNS, hence on top of specifying the service discovery `type`, you need to specify `port` in your configuration. 
```

Comment:
I would mention the type of records it currently reads.

Reply:
added